src/UI/Buyer/src/app/profile/containers/profile/profile.component.ts

```diff
@@ -30,4 +37,13 @@ export class ProfileComponent implements OnInit {
   selectTab(tab: ProfileTab): void {
     this.selectedTab = tab.display;
   }
-}
+
+  logout() {
```

Comment:
I think this was kind of a holdover from angular 1 app where rob had stored other cookies not related to the ordercloud sdk. The convention was to prefix any of those cookies with the appname so that on logout we could remove any app-specific cookies from the client's computer. 

I'm ok just using the ocTokenService since we don't have any other cookies in there anyways.

Reply:
but yes, lets keep one log out method

Reply:
So then do you want me to update app-auth.service with this?
```javascript
 this.ocTokenService.RemoveAccess();
    if (this.appStateService.isAnonSubject.value) {
      this.baseResolveService.resetUser();
    } else {
      this.router.navigate(['/login']);
    }
```

Reply:
If you're willing to take this on, the function you add to app-auth.service will have to handle all the cases of both versions. It will probably use `this.ocTokenService.RemoveAccess()` and might return a type of `Observable<something>`.